### PR TITLE
fix(dev): align parameter name with options name

### PR DESCRIPTION
**Pull request by geigerzaehler**

### CODE REVIEW

**Review by rudolfs — APPROVED** [review]

**Comment by rudolfs** [comment]
While you're at it:
```diff
diff --git a/DEVELOPMENT.md b/DEVELOPMENT.md
index d5e43005..8987c15f 100644
--- a/DEVELOPMENT.md
+++ b/DEVELOPMENT.md
@@ -204,9 +204,6 @@ Here's a list of all scripts that are intended for developer use:
     yarn dist                   # Build an installable Upstream package for the
                                 # current platform

-    yarn release                # Start a two-step process to cut a new
-                                # release, see DEVELOPMENT.md for more details
-
     yarn typescript:check       # Type-check all UI *.ts and *.svelte files
     yarn prettier:check         # Check UI code formatting
     yarn prettier:write         # Auto-format UI code
@@ -518,7 +515,7 @@ All Github access tokens _must_ have the `public_repo` scope.
     2. Publish the CI artifacts as release candidate binaries.

        ```bash
-       ./sripts/release.ts publish-rc-binaries
+       ./scripts/release.ts publish-rc-binaries
        ```
```

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -518,43 +515,44 @@ All Github access tokens _must_ have the `public_repo` scope.
...
        ```
 
     4. Test the release by walking through the QA issues.
     5. (Optional) To fix bugs, create a PR with the fixes based on the release
        candidate branch. Once it has been approved, squash merge it into the
-       release candidate  branch (see [“Merging Pull Requests"][merging-prs])
-       and continue QA.
+       release candidate branch (see [“Merging Pull Requests"][merging-prs]).
+       Then restart the “Test the release” step. (Skip creating a QA
+       issue in 2.2.)
```
Comment (rudolfs) [inline DEVELOPMENT.md:533]:
```suggestion
       issue in 2.3.)
```

Comment (geigerzaehler) [reply DEVELOPMENT.md]: Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1948/commits/2aa4bd017e979991dbb9825e9bd95e2880dcb1ee

**Comment by geigerzaehler** [comment]
Could you take another look @rudolfs?

**Comment by rudolfs** [comment]
Looks good!

**Pull request merged**